scripts/framework-applications/suspend-suppliers-without-agreements.py

```diff
@@ -88,25 +102,31 @@
     supplier_ids = get_supplier_ids_from_args(args)
     suppliers = find_suppliers_without_agreements(client, framework_slug, supplier_ids)
 
-    with open(output_dir / FILENAME, 'w') as csvfile:
-        csv_headers = ['Supplier email', 'Supplier ID', "No. of services suspended"]
+    framework_name = framework["name"]
+    contract_title = get_framework_contract_title(FRAMEWORKS_PATH, framework_slug)
 
-        writer = csv.writer(csvfile, quoting=csv.QUOTE_MINIMAL)
-        writer.writerow(csv_headers)
-
-        for supplier in suppliers:
-            supplier_id = supplier["supplierId"]
-            framework_info = client.get_supplier_framework_info(supplier_id, framework_slug)
+    for supplier in suppliers:
+        supplier_id = supplier["supplierId"]
+        framework_info = client.get_supplier_framework_info(supplier_id, framework_slug)
 
+        logger.info(f"{prefix}Suspending services for supplier {supplier_id}")
+        if not dry_run:
             # Do the suspending
             suspended_service_count = suspend_supplier_services(
                 client, logger, framework_slug, supplier_id, framework_info, dry_run
             )
-
-            if suspended_service_count == 0:
-                # We should have logged why already
-                continue
-
-            # Compile a list of email addresses for the supplier (to be sent via Notify) and add to the CSV
-            for supplier_email in get_all_email_addresses_for_supplier(client, framework_info):
-                writer.writerow([supplier_email, supplier_id, suspended_service_count])
+            logger.info(f"{prefix}Suspended {suspended_service_count} services for supplier {supplier_id}")
```

Comment:
Do we only want to log this where `suspended_service_count` is >0?

Reply:
I think we also want to log a warning if we do somehow suspend 0 services, as that means something's gone wrong